packages/terra-layout/tests/wdio/layout-slide-panel-spec.js

```diff
@@ -1,33 +1,32 @@
 /* globals $ */
 Terra.describeViewports('Toggle the slide panel and hidden styles', ['medium'], () => {
   before(() => {
-    browser.url('/#/raw/tests/terra-layout/layout/layout-slide-panel-example');
+    browser.url('/raw/tests/terra-layout/layout/layout-slide-panel-example');
     browser.waitForExist('[class*=_panel_][aria-hidden="true"]');
   });
-
-  Terra.it.matchesScreenshot({ selector: '#site' });
+  it('Toggles the slide panel and hidden styles', () => {
+    Terra.validates.element('default', { selector: '#site' });
+  });
 });
 
 describe('Toggle the slide panel click', () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-layout/layout/layout-slide-panel-example');
+  it('Toggles the slide panel click', () => {
+    browser.url('/raw/tests/terra-layout/layout/layout-slide-panel-example');
     browser.click('#test-toggle-1');
     browser.waitForExist('[class*=_panel_][aria-hidden="false"]');
+    Terra.validates.element('toggle click', { selector: '#site' });
   });
-
-  Terra.it.matchesScreenshot({ selector: '#site' });
 });
 
 describe('Toggle the slide panel double click', () => {
```

Comment:
all three it blocks toggle the same example. we could remove the inner describe blocks and have the it blocks directly under `Terra.describeViewports`. is that out of scope for this pr? what are your thoughts?

Reply:
First test is for medium viewport and the other 2 are for huge, so I think thats the reason why they are outside the `Terra.describeViewports`.